### PR TITLE
Support alternative images by locale for programme types

### PR DESCRIPTION
**Pull request by iant**

### CODE REVIEW

File: app/models/programme_type.rb [hunk 1]
```diff
@@ -15,11 +15,13 @@
 class ProgrammeType < ApplicationRecord
   extend Mobility
   include TransifexSerialisable
+  include TranslatableAttachment
```
Comment (iant) [inline app/models/programme_type.rb:18]: thank you for putting together this concern @urbanwide 🙏  (nice implementation too!)  made implementing the multi file uploads super simple :)

**Review by urbanwide — APPROVED** [review]
Looks good @iant 👍

File: app/models/programme_type.rb (hunk 1 above)
Comment (urbanwide) [reply app/models/programme_type.rb:18]: Yay, glad it has been useful :-)

Comment (iant) [reply app/models/programme_type.rb:18]: it really was!  now I just need to do the same for activity_categories, activity types, intervention type groups, and intervention types :D

Comment (urbanwide) [reply app/models/programme_type.rb:18]: Hehe good luck :-)

**Pull request merged**